CONTRIBUTIONS.md

```diff
@@ -0,0 +1,145 @@
+# Contributing 
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to all the repositories managed by Jordan Harband, which are hosted on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+# Table of Contents 
+
+[How Can I contribute?](#how-can-i-contribute)
+
+* [Resolving Existing Issues](#resolving-existing-issues)
+
+* [Reporting Issues](#reporting-issues)
+
+* [Documentation](#documentation)
+
+[Dev Environment](#dev-environment)
+
+[Style Guide / Coding conventions](#style-guide--coding-conventions)
+
+* [Pull Requests](#pull-requests)
+
+* [How to Write Commit Messages](#commit-messages)
+
+[Code of Conduct](#code-of-conduct)
+
+[Where can I ask for help?](#where-can-i-ask-for-help)
+
+# How Can I Contribute?
+
+There are lots of ways to get involved. Here are some suggestions of things we'd love some help with. 
+
+## Resolving existing issues 
+
+You can consider helping out with issues already requiring attention - look for a "help wanted" label. 
+
+## Reporting issues 
+
+If you run into problems in the project, you can report them by opening a new issue within the repository. Before filing an issue, please perform a cursory search to see if the problem has already been reported. If it has **and the issue is still open**, add a comment to the existing issue instead of opening a new one.
+
+### How Do I Submit a (Good) Bug Report? 
```

Comment:
let's add something like "if there's an issue template, please always fill it out in its entirety - the template is there for a reason"